planemo/commands/cmd_workflow_test_init.py

```diff
@@ -18,29 +18,35 @@
 @options.required_workflow_arg()
 @options.force_option()
 @options.workflow_output_artifact()
+@options.split_job_and_test()
 @command_function
-def cli(ctx, workflow_path, output=None, **kwds):
+def cli(ctx, workflow_path, output=None, split_test=False, **kwds):
     """Initialize a Galaxy workflow test description for supplied workflow.
 
     Be sure to your lint your workflow with ``workflow_lint`` before calling this
     to ensure inputs and outputs comply with best practices that make workflow
     testing easier.
     """
-    # TODO: implement force
     path_basename = os.path.basename(workflow_path)
     job = job_template(workflow_path)
     if output is None:
         output = new_workflow_associated_path(workflow_path)
     job_output = new_workflow_associated_path(workflow_path, suffix="job1")
-    if not can_write_to_path(job_output, **kwds) or not can_write_to_path(output, **kwds):
+    if not can_write_to_path(output, **kwds):
         ctx.exit(1)
 
     test_description = [{
          'doc': 'Test outline for %s' % path_basename,
-         'job': os.path.basename(job_output),
-         'outputs': output_stubs_for_workflow(workflow_path)
+         'job': job,
+         'outputs': output_stubs_for_workflow(workflow_path),
     }]
+    if split_test:
+        job_output = new_workflow_associated_path(workflow_path, suffix="job1")
+        if not can_write_to_path(job_output, **kwds):
+            ctx.exit(1)
```

Comment:
Does that print some information about why this failed ?

Reply:
I think so:

``error("%s already exists, exiting." % path)``

Reply:
It'd be good to also check the directory exists and is writable... but yeah...